tests/Umbraco.Tests.UnitTests/Umbraco.Infrastructure/Services/PropertyValidationServiceTests.cs

```diff
@@ -279,6 +287,30 @@ public void Validate_Properties_On_All()
         Assert.AreEqual(4, invalid.Length);
     }
 
+    [TestCase(null, true)]
+    [TestCase(24, true)]
+    [TestCase("test", true)]
+    [TestCase("{\"test\": true}", true)]
+    public void ValidatePropertyValue_Returns_Expected_Result_For_Missing_Editor(object? value, bool expectedResult)
```

Comment:
`expectedResult` is `true` for all cases... is that on purpose, or are we missing some test cases here?

Reply:
Hm. The gist of this PR is to perform a fallback to the missing data editor if a concrete data editor can't be found. Unless I'm terribly mistaken, that means we can't perform any kind of validation for the property value. So I don't think we can do much about it? Maybe just remove `expectedResult` altogether.